Sources/Baggage/BaggageContext.swift

```diff
@@ -41,11 +41,21 @@ public struct BaggageContext {
         get {
             self._storage[ObjectIdentifier(key)]?.forceUnwrap(key)
         } set {
-            self._storage[ObjectIdentifier(key)] = newValue == nil ? nil : ValueContainer(value: newValue!)
+            self._storage[ObjectIdentifier(key)] = newValue == nil ? nil : ValueContainer(keyName: String(describing: key.self), value: newValue!)
```

Comment:
`!`-alert! ⚠️ 

You can avoid it by doing:

```swift
            self._storage[ObjectIdentifier(key)] = newValue.map {
                ValueContainer(keyName: String(describing: key.self), value: $0)
            }
```

I would also propose:

```suggestion
            self._storage[ObjectIdentifier(key)] = newValue.map {
                // TODO: consider if keys should allow optional string names; static property on the type?
                ValueContainer(keyName: String(describing: key.self), value: $0)
            }
```

since we may consider that